<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>444</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QToolBox" name="toolBox">
    <property name="currentIndex">
     <number>6</number>
    </property>
    <widget class="QWidget" name="pageApp">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>266</width>
       <height>249</height>
      </rect>
     </property>
     <attribute name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/images/802.bmp</normaloff>:/images/images/802.bmp</iconset>
     </attribute>
     <attribute name="label">
      <string>QCoreApplication类</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>applicationDirPath()</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>applicationFilePath()</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>applicationName()</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>setApplicationName</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="toolTip">
            <string>lib</string>
           </property>
           <property name="text">
            <string>libraryPaths()</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>organizationName()</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="text">
            <string>exit()退出程序</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="pageFile">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>266</width>
       <height>276</height>
      </rect>
     </property>
     <attribute name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/images/804.bmp</normaloff>:/images/images/804.bmp</iconset>
     </attribute>
     <attribute name="label">
      <string>QFile类</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>静态函数</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_14">
           <property name="text">
            <string>copy()</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_15">
           <property name="text">
            <string>exists()</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_16">
           <property name="text">
            <string>remove()</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_17">
           <property name="text">
            <string>rename()</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_18">
           <property name="text">
            <string>move To Trash()</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>成员函数</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_19">
           <property name="text">
            <string>copy()</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_20">
           <property name="text">
            <string>exists()</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_21">
           <property name="text">
            <string>remove()</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_22">
           <property name="text">
            <string>rename()</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_23">
           <property name="text">
            <string>move To Trash()</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="pageFileInfo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-74</y>
       <width>290</width>
       <height>324</height>
      </rect>
     </property>
     <attribute name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/images/135.JPG</normaloff>:/images/images/135.JPG</iconset>
     </attribute>
     <attribute name="label">
      <string>QFileInfo类</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>文件信息</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_24">
           <property name="text">
            <string>absoluteFilePath()</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_25">
           <property name="text">
            <string>absolutePath()</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_26">
           <property name="text">
            <string>fileName()</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_27">
           <property name="text">
            <string>filePath()</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_28">
           <property name="text">
            <string>size()</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_29">
           <property name="text">
            <string>path()</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pushButton_40">
           <property name="text">
            <string>baseName()</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="pushButton_41">
           <property name="text">
            <string>completeBaseName()</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="pushButton_30">
           <property name="text">
            <string>suffix()</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="pushButton_31">
           <property name="text">
            <string>completeSuffix()</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="pushButton_32">
           <property name="text">
            <string>isDir()</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="pushButton_33">
           <property name="text">
            <string>isFile()</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="pushButton_34">
           <property name="text">
            <string>isExecutable()</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QPushButton" name="pushButton_35">
           <property name="text">
            <string>birthTime()</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QPushButton" name="pushButton_36">
           <property name="text">
            <string>lastModified()</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QPushButton" name="pushButton_37">
           <property name="text">
            <string>lastRead()</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QPushButton" name="pushButton_38">
           <property name="text">
            <string>静态函数exist()</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QPushButton" name="pushButton_39">
           <property name="text">
            <string>exists()</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="pageDir">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-204</y>
       <width>266</width>
       <height>468</height>
      </rect>
     </property>
     <attribute name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/images/007.GIF</normaloff>:/images/images/007.GIF</iconset>
     </attribute>
     <attribute name="label">
      <string>QDir类</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="title">
         <string>静态函数</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_42">
           <property name="text">
            <string>tempPath</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_43">
           <property name="text">
            <string>rootPath()</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_44">
           <property name="text">
            <string>homePath()</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_45">
           <property name="text">
            <string>drives()</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_46">
           <property name="text">
            <string>currentPath()</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_47">
           <property name="text">
            <string>setCurrent()</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_8">
        <property name="title">
         <string>文件或目录操作</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_52">
           <property name="text">
            <string>setPath()</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_50">
           <property name="text">
            <string>remove()</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_49">
           <property name="text">
            <string>rmdir()</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_48">
           <property name="text">
            <string>mkdir()</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_51">
           <property name="text">
            <string>rename()</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_53">
           <property name="text">
            <string>removeRecuesively()</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_9">
        <property name="title">
         <string>文件或目录信息</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_54">
           <property name="text">
            <string>absoluteFilePath()</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_55">
           <property name="text">
            <string>absolutePath()</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_56">
           <property name="text">
            <string>canonicalPath()</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_57">
           <property name="text">
            <string>filePath()</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_58">
           <property name="text">
            <string>path()</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_59">
           <property name="text">
            <string>dirName()</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pushButton_60">
           <property name="text">
            <string>exists()</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="pushButton_61">
           <property name="text">
            <string>isEmpty()</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="pushButton_62">
           <property name="text">
            <string>entryList(dir)</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="pushButton_63">
           <property name="text">
            <string>entryList(file)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="page_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>280</width>
       <height>234</height>
      </rect>
     </property>
     <attribute name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/images/806.bmp</normaloff>:/images/images/806.bmp</iconset>
     </attribute>
     <attribute name="label">
      <string>QTemporaryDir类</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_8">
           <property name="text">
            <string>1.在系统临时目录下创建临时文件夹</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_9">
           <property name="text">
            <string>2.在指定目录下创建临时文件夹</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_10">
           <property name="text">
            <string>3.在当前目录下创建临时文件夹</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="page_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>280</width>
       <height>234</height>
      </rect>
     </property>
     <attribute name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/images/174.JPG</normaloff>:/images/images/174.JPG</iconset>
     </attribute>
     <attribute name="label">
      <string>QTemporaryFile类</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QGroupBox" name="groupBox_10">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QPushButton" name="pushButton_64">
           <property name="text">
            <string>在系统临时目录下创建临时文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_65">
           <property name="text">
            <string>在指定目录下创建临时文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_66">
           <property name="text">
            <string>在当前目录下创建临时文件</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="page">
     <attribute name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/images/images/714.bmp</normaloff>:/images/images/714.bmp</iconset>
     </attribute>
     <attribute name="label">
      <string>QFileSystemWatcher类</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <item>
       <widget class="QGroupBox" name="groupBox_11">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QPushButton" name="pushButton_67">
           <property name="text">
            <string>addPath()并开始监视</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_68">
           <property name="text">
            <string>removePath()并停止监视</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_69">
           <property name="text">
            <string>directories()</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_70">
           <property name="text">
            <string>files()</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_11">
         <property name="text">
          <string>打开文件</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/images/images/007.GIF</normaloff>:/images/images/007.GIF</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_12">
         <property name="text">
          <string>打开目录</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/images/images/122.bmp</normaloff>:/images/images/122.bmp</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_13">
         <property name="text">
          <string>清除文本框</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/images/images/212.bmp</normaloff>:/images/images/212.bmp</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>文件</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editFile"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>目录</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editDir"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>信息</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
